bazel/repository_locations.bzl

```diff
@@ -279,11 +279,11 @@ REPOSITORY_LOCATIONS_SPEC = dict(
         project_name = "libipp-crypto",
         project_desc = "Intel® Integrated Performance Primitives Cryptography",
         project_url = "https://github.com/intel/ipp-crypto",
-        version = "2021.4",
-        sha256 = "23e250dcf281aa00d186be8dc4e34fa8fc5c95a0895694cd00b33f18af5d60c7",
+        version = "2021.5",
+        sha256 = "0b277548c59e6bfe489e634d622b54be3708086fc006a441d39922c2d6d43f0d",
         strip_prefix = "ipp-crypto-ippcp_{version}",
         urls = ["https://github.com/intel/ipp-crypto/archive/ippcp_{version}.tar.gz"],
-        release_date = "2021-10-01",
+        release_date = "2021-12-22",
```

Comment:
i think the utc date must be `2021-12-21`

https://dev.azure.com/cncf/envoy/_build/results?buildId=97278&view=logs&j=1c04237c-5f52-5959-5f83-5522b3556331&t=db3f3889-c922-566b-fa2e-4bc4948bc467&l=319